fuel/app/classes/presenter/welcome/hello.php

```diff
@@ -16,6 +16,13 @@ class Presenter_Welcome_Hello extends Presenter
 	 */
 	public function view()
 	{
-		$this->name = $this->request()->param('name', 'World');
+    $hogehogedhoge="fugafugafuga";
+    $hogehoddgehoge="fugafugafuga";
+    $hogdddehogehoge="fugafugafuga";
+    $dddhogehogehoge="fugafugafuga";
+
+
+    $hogehoge = "jdoaisjglkl,.,m.,m.,m.";
```

Comment:
__[code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed
* String "jdoaisjglkl,.,m.,m.,m." does not require double quotes; use single quotes instead